pkg/releaselock/controller.go

```diff
@@ -75,12 +91,19 @@ func NewLockReleaseController(client kubernetes.Interface, config *LockReleaseCo
 	}
 	// Add a uniquifier so that two processes on the same host don't accidentally both become active.
 	id := hostname + "_" + string(uuid.NewUUID())
+	ratelimiter := workqueue.NewMaxOfRateLimiter(
+		workqueue.NewItemExponentialFailureRateLimiter(5*time.Millisecond, 1000*time.Second),
```

Comment:
create flags for the failure rate limiter args?

Reply:
done